lib/features/parkings_view/widgets/parking_wide_tile_card.dart

```diff
@@ -143,3 +131,69 @@ class _RightColumn extends StatelessWidget {
     );
   }
 }
+
+class _FavouriteWidget extends StatefulWidget {
```

Comment:
So my first note is that this can go to separate file as it's pretty much separate concept